pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -2539,7 +2539,7 @@ async def core_pick_up_resource(
     """
 
     # Get center of source plate. Also gripping height and plate width.
-    center = resource.get_absolute_location() + resource.center() + offset
+    center = resource.get_absolute_location(x="c", y="c", z="b") + offset
```

Comment:
moving `resource.center()` into `get_absolute_location` allows us to apply the correct rotation matrix